scripts/kani-regression.sh

```diff
@@ -8,6 +8,17 @@ fi
 set -o pipefail
 set -o nounset
 
+ARG=
+# Check for optional arguments
+if (( "$#" > 0 )); then
```

Comment:
Do we expect developers to run these tests locally? I would prefer if this script always run the same set of tests.

Reply:
I think so (at least for debugging purposes). I can add this to a separate script, although it fitted nicely within this script since all the setup steps are handled.